src/main/java/io/orkes/conductor/client/automator/TaskRunner.java

```diff
@@ -89,8 +89,17 @@ class TaskRunner {
         this.domain = Optional.ofNullable(PropertyFactory.getString(taskType, DOMAIN, null))
                 .orElseGet(() -> Optional.ofNullable(PropertyFactory.getString(ALL_WORKERS, DOMAIN, null))
                         .orElse(taskToDomain.get(taskType)));
-        this.errorAt = PropertyFactory.getInteger(taskType, DOMAIN, 100);
 
+        int defaultLoggingInterval = 100;
+        int errorInterval = PropertyFactory.getInteger(taskType, "LOG_INTERVAL", 0);
+        if(errorInterval == 0) {
+            errorInterval = PropertyFactory.getInteger(ALL_WORKERS, "LOG_INTERVAL", 0);
+        }
+        if(errorInterval == 0) {
+            errorInterval = defaultLoggingInterval;
+        }
+        this.errorAt = errorInterval;
+        LOGGER.info("Polling errors will be sampled at every {} error (after the first 100 errors) for taskType {}", this.errorAt, taskType);
```

Comment:
So if we default to `defaultLoggingInterval`, then this log will print - 
`Polling errors will be sampled at every 100 error (after the first 100 errors) for taskType T`.
 
Where does the `after the first 100 errors` logic kick in? Maybe, I am not understanding something here.


Another minor thing -  In `every {} error`, I think we can say `errors`